pkg/websocket/client.go

```diff
@@ -437,8 +437,18 @@ func (c *Client) writePump() {
 				if ws.IsUnexpectedCloseError(err, ws.CloseNormalClosure) {
 					c.cfg.Log.Error("write error: ", err)
 				}
-				c.notifyClose <- err
 
+				// writing to notifyClose during
+				select {
+				case c.notifyClose <- err:
+					c.cfg.Log.WithFields(log.Fields{
+						"prefix": "websocket.Client.writePump",
+					}).Debug("ping notifyClose")
```

Comment:
That might get confusing for someone trying to debug logs without context. a message like "No response from ping, closing connection"

and (below):
"no response from ping because connection seems to be closing" or something like that.

Reply:
Updated to "Failed to send ping; closing connection" and "Failed to send ping; connection is resetting".